app/Http/Controllers/UserController.php

```diff
@@ -59,6 +60,16 @@ public function post_completed_task(Request $request)
         $user_id = $request->user()->id;
         $task_id = $request->task_id;
 
+        // ユーザが投票したかどうかの情報を取得
+        $is_voted = Vote::select()
+        ->where('user_id','=',$user_id)
+        ->exists();
+        
+        // 投票していたらチェックのオンオフができない
+        if ($is_voted) {
+            return response()->json('Request Denied', Response::HTTP_BAD_REQUEST);
```

Comment:
```suggestion
            return response()->json('already voted of junbo takara-kuji', Response::HTTP_BAD_REQUEST);
```
の様な具体的なメッセージが良いと思います。